<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>225</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Korad PSU</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="oVset1">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>--.--</string>
    </property>
   </widget>
   <widget class="QLabel" name="oIset1">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>127</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-.---</string>
    </property>
   </widget>
   <widget class="QLabel" name="oVout">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>220</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Digital-7</family>
      <pointsize>70</pointsize>
     </font>
    </property>
    <property name="text">
     <string>--.--</string>
    </property>
   </widget>
   <widget class="QLabel" name="oIout">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>220</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Digital-7</family>
      <pointsize>70</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-.---</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>308</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="oStatusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
